src/code/autolaunch/autolaunch.ts

```diff
@@ -171,8 +171,8 @@ export default function autolaunchInteractive() {
     var launchParams = {url: interactiveData.interactiveStateUrl, source: documentId, collaboratorUrls: interactiveData.collaboratorUrls}
     // If there is a linked state and no interactive state then change the source document to point to the linked recordid and add the access key.
     if (stateValid(linkedState) && !interactiveStateAvailable) {
-      launchParams.source = linkedState.docStore.recordid
-      launchParams.readOnlyKey = linkedState.docStore.accessKeys.readOnly
+      launchParams.source = linkedState.docStore.recordid;
```

Comment:
The inconsistency of the no-semicolons convention is one of the things that annoys me about it.